backend/src/modules/player-profiles/core/application/event/SendEmailAfterPlayerProfileWasCreatedEventHandler.ts

```diff
@@ -0,0 +1,23 @@
+import { EventHandler } from '../../../../../shared/core/application/event/EventHandler';
+import { CommandPublisher } from '../../../../../shared/core/application/command/CommandBus';
+import { PlayerProfileWasCreated } from '../../domain/event/PlayerProfileWasCreated';
+import { SendEmail } from '../../../../email-sending/core/application/command/SendEmail';
+
+export class SendEmailAfterPlayerProfileWasCreatedEventHandler implements EventHandler<PlayerProfileWasCreated> {
+  constructor(private readonly commandPublisher: CommandPublisher) {}
+
+  async handle(event: PlayerProfileWasCreated): Promise<void> {
+    await this.commandPublisher.execute(
+      new SendEmail({
+        emailAddress: event.emailAddress,
+        subject: `Welcome on board ${event.firstName}!`,
+        htmlContent: `
+        <div style="text-align: center">
```

Comment:
Fajna zamiana na HTMLa :)

Reply:
no wgl zastanawiałem się czy nie dać jakiegoś tła i wgl, ale to najwyżej kiedyś ... szkoda czasu chyba teraz na to, a przynajmniej widać, że można coś na takiego ładnie wstawić